sdk_extensions/zpages/src/test/java/io/opentelemetry/sdk/extensions/zpages/ZPageHttpHandlerTest.java

```diff
@@ -29,14 +29,19 @@
   @Test
   void parseEmptyQuery() throws URISyntaxException, UnsupportedEncodingException {
     URI uri = new URI("http://localhost:8000/tracez");
+    String queryString = "";
     assertThat(ZPageHttpHandler.parseQueryMap(uri)).isEmpty();
+    assertThat(ZPageHttpHandler.parseQueryMap(queryString)).isEmpty();
   }
 
   @Test
   void parseNormalQuery() throws URISyntaxException, UnsupportedEncodingException {
     URI uri =
         new URI("http://localhost:8000/tracez/tracez?zspanname=Test&ztype=1&zsubtype=5&noval");
+    String queryString = "zspanname=Test&ztype=1&zsubtype=5&noval";
     assertThat(ZPageHttpHandler.parseQueryMap(uri))
```

Comment:
Is this saying that using the GET request still works?

Reply:
TraceZ still uses GET request for displaying information

Reply:
Thinking of this now, GET request can still work on TraceConfigZ page, I should try to limit to POST request only